cuebot/src/main/resources/opencue.properties

```diff
@@ -5,22 +5,22 @@ spring.velocity.checkTemplateLocation=false
 # datasource.cueDataSource.driverClassName=oracle.jdbc.OracleDriver
 # datasource.cueDataSource.url=jdbc:oracle:oci:@dbname
 
-datasource.cueDataSource.driverClassName=org.postgresql.Driver
-datasource.cueDataSource.url=jdbc:postgresql://dbhost/dbname
-datasource.cueDataSource.username=cue
-datasource.cueDataSource.password=password
+datasource.cue-data-source.driver-class-name=org.postgresql.Driver
```

Comment:
Is there some requirement to change these names? This seems like it will break existing Cuebot deployments, who may be specifying these properties via commandline flags.

If it has to happen we will need to do it carefully, make sure we update docs, etc.

Reply:
The "cueDataSource" style naming was deprecated a while back, now its not supported by spring anymore and the app refuses to start without the proper property naming.

Reply:
Ok, understood. We will want to accompany submitting and/or releasing this change with a coordinated docs change.